config/constants.ts

```diff
@@ -76,7 +76,7 @@ export const rdsMasterSecretName = 'orcabus/master-rds'; // pragma: allowlist se
 validateSecretName(rdsMasterSecretName);
 
 // Other constants that exist in the SharedStack
-export const regName = 'OrcaBusSchemaRegistry';
+export const regName = 'orcabus.events';
```

Comment:
I don't think we have other registries planned, but can we be a bit more specific?
E.g. `eventRegName` or `schemaRegName`

Reply:
Ok, I will change it to `schemaRegistryName`

Reply:
Fixed with https://github.com/umccr/orcabus/commit/efd9a5b5fac29a669c8d34e7e4b49dd692585f49